src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -220,23 +186,26 @@
           :container-style style/section-label-right}]])
      [rn/view {:style style/routes-inner-container}
       [render-network-values
-       {:token-symbol              token-symbol
-        :network-values            sender-network-values
-        :on-press                  #(disable-chain %1
-                                                   disabled-from-chain-ids
-                                                   token-available-networks-for-suggested-routes)
-        :to?                       false
-        :theme                     theme
-        :loading-suggested-routes? loading-suggested-routes?}]
+       {:token-symbol                              token-symbol
+        :network-values                            sender-network-values
+        :on-press                                  #(disable-chain
+                                                     %1
```

Comment:
maybe it's clearer to use fn here rather than # and explain the var use?

Reply:
Updated!